tests/pages/api/repair-request/create.test.ts

```diff
@@ -86,4 +87,51 @@ describe("POST /api/repair-request", () => {
       }
     });
   });
-});
+
+  it("should be able to recieve an email", async () => {
```

Comment:
Alternatively, just check that the function is called using a spy or something as mentioned above

Reply:
I've improved the test to:

1. Run the test only in the mock environment; otherwise, skip it. And use the same environment variable to determine the endpoint in the SES client.
2. Get the mock email results from localhost:8085/store, which is the endpoint where the mock SES email service stores its mock results.
3. Check the mock results against the expected ones.